backend/tests/apps/mentorship/api/internal/nodes/test_api_internal_program.py

```
@@ -0,0 +1,129 @@
+        status=ProgramStatusEnum.COMPLETED,
+    )
+    assert update_input.domains is None
+    assert update_input.admin_logins is None
+    assert update_input.tags is None
+
+
+def test_update_program_status_input_fields():
+    """Test that UpdateProgramStatusInput fields are correctly defined."""
+    assert UpdateProgramStatusInput.__annotations__["key"] == str
+    assert UpdateProgramStatusInput.__annotations__["name"] == str
+    assert UpdateProgramStatusInput.__annotations__["status"] == ProgramStatusEnum
```

Ruff (0.14.10)

No newline at end of file

Add trailing newline

(W292)